chat-plugins/info.js

```diff
@@ -460,6 +288,15 @@ var commands = exports.commands = {
 		}
 		this.sendReply(buffer);
 	},
+	datahelp:
+		"/data [pokemon/item/move/ability] - Get details on this pokemon/item/move/ability/nature." + "\n" +
+		"!data [pokemon/item/move/ability] - Show everyone these details. Requires: + % @ & ~",
+	detailshelp:
+		"/details [pokemon] - Get additional details on this pokemon/item/move/ability/nature." + "\n" +
+		"!details [pokemon] - Show everyone these details. Requires: + % @ & ~",
+	dthelp:
```

Comment:
I don't really like how this has to be written twice, but I'm not sure it's practical to try to map it to detailshelp. The only method I could think of would be to make it a different data type (probably an object like `{'alias':'detailshelp'}`, but that looked ugly and is inconsistent with how command aliases work. I don't think this is really an issue at the moment, but thinking of a way to have help aliases would future-poof `/help`.